production/helm/loki/CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ Entries should include a reference to the pull request that introduced the chang
 
 ## 6.19.0
 
+- [CHANGE] Changed version of Minio helm chart to 5.3.0
```

Comment:
can you move this entry up above into the unreleased section? we automatically release weekly now, so this will be included in the right version automatically if placed in the unlreased section. thanks!

Reply:
I moved it up and added the PR number.